docs/TESTING.md

```diff
@@ -0,0 +1,150 @@
+---
+For release: v1.0
+---
+
+# Test Plan
+
+## Introduction
+
+The idea of this The Test Plan is to communicate the test approach to team members. 
+
+As the team works with the product, they will define the needs for the vNext.
+
+### Team communication
+
+Perform communication through Slack and Github issues.
+
+### Objectives
+
+The 1.0 phase will focus on manual testing for now, and then we'll migrate to automation in the future. Preferably in the next major test plan update, the next app release will follow.
+
+App milestones for 1.0:  
+[https://github.com/WorldHealthOrganization/app/milestone/4](https://github.com/WorldHealthOrganization/app/milestone/4)
+
+### Application architecture
+
+Application is build using Flutter framework. We need to have this in mind and choose appropriate tools and approach.
+
+We're supporting Android and iOS mobile operating systems.
+
+* IOS from 8 to 13
+* Android from 4.1\(API 16\) to 10.0 \(API 29\);
+
+## Plan
+
+### Manual tests
+
+#### Scope
+
+In manual testing, we should focus on all user interface elements, hand washing animation in "Protect Yourself".
+
+#### Examples of configurations to focus on;
```

Comment:
Can we get some market data on this?
Maybe by WHO region?  What is the distribution of device types per region?

Reply:
Yes, that will be in the device matrix sheet.
I'm collecting data mostly from these resources:
https://gist.github.com/pwicherski/63336437f9f3cacb92d5b5c67427e4a8